{{cookiecutter.repo_name}}/README.md

```diff
@@ -0,0 +1,15 @@
+# PythonTemplate
+
+This is an example of a python package / collection of scripts for Nieuwe Warmte Nu teams.
+Use this as a template for starting your project. 
+
```

Comment:
We should probably replace this with

```
# {{cookiecutter.project_name}}

{{cookiecutter.project_short_description}}
```
so the readme just contains the project name and description.

Reply:
Done